src/matter/cluster/PressureMeasurementCluster.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, OptionalAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8 } from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for Pressure Measurement.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 2.4
+ */
+export const PressureMeasurementCluster = Cluster({
+    id: 0x0403,
+    name: "PressureMeasurement",
+    revision: 3,
+    features: {
+        extended: BitFlag(0) // The cluster is capable of extended range and resolution
```

Comment:
Reformat this as a JsDoc:
```
/** The cluster is capable of extended range and resolution */
extended: BitFlag(0),
```

VisualCode, and probably other IDEs, will magically displays this jsdoc when the flags are getting defined (on "extended: true" statement).

Reply:
I need to figure out how to get "vi" to understand jsdoc :-)

Fixed this and the comments below

Reply:
I strongly recommend Visual Code for Typescript development :-)
For once a Microsoft product that doesn't suck, and is free and cross-platform!

Reply:
I also agree ... VS Code (free) or WebsStorm (not free) is really good and helps a lot